tests/test_controller.py

```diff
@@ -16,6 +16,10 @@ def make_snippet(ident, code):
 snippet_lipsum = make_snippet('lipsum', code="Lorem ipsum dolor sit amet")
 snippet_python = make_snippet('egg', code='print("Hello world")')
 snippet_htmlxss = make_snippet('htmlxss', code='<script>alert("XSS");</script>')
+snippet_classified = make_snippet('classified', code='T_xJV3P^FcvYijzH')
+
+UA_HUMAN = "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:85.0) Gecko/20100101 Firefox/85.0"
+UA_BOT = "Discordbot/2.0"
```

Comment:
Je pense qu'on peut faire comme UA_HUMAN et donner un User-Agent complet, étant donné qu'il s'agit de tester la regex
```suggestion
UA_BOT = "Mozilla/5.0 (compatible; Discordbot/2.0; +https://discordapp.com)"
```